docs/rm_to_eq_runner_payload_v2.md

```diff
@@ -30,12 +30,13 @@ The following metadata properties are always required for the EQ Runner, they do
 
 The schema selection field determine the mechanism used by EQ Runner to load the questionnaire schema JSON.
 
-The schema used by an EQ Runner can be selected one of two ways.
+The schema used by an EQ Runner can be selected one of three ways.
```

Comment:
Should we explicitly say that they are mutually exclusive? Have no strong opinions just a question.

Reply:
Can do if you feel it makes it clearer? 👍

Reply:
Not. too much of a difference for me. Resloving 👍